testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/hibernate/SFSBHibernate2LcacheStats.java

```diff
@@ -79,6 +79,7 @@ public void setupConfig() {
             Configuration configuration = new Configuration().setProperty(AvailableSettings.USE_NEW_ID_GENERATOR_MAPPINGS,
                     "true");
             configuration.getProperties().put(AvailableSettings.JTA_PLATFORM, JBossAppServerJtaPlatform.class);
+            configuration.getProperties().put(AvailableSettings.TRANSACTION_COORDINATOR_STRATEGY, "jta");
```

Comment:
tl;dr

https://hibernate.atlassian.net/browse/HHH-11024?focusedCommentId=82236&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-82236 

I need to find the above referenced prior conversation.

https://github.com/hibernate/hibernate-orm/blob/5.0/migration-guide.adoc#transactions mentions the change for ORM 5.0/5.1.  We are currently using Hibernate ORM 5.0.10 with EAP 7.0

https://issues.jboss.org/browse/JBEAP-5768 (EAP doc change jira for "Non-JPA applications that use Hibernate with JTA must set hibernate.transaction.coordinator_class") mentions that the ORM 5.0 doc change was via commit https://github.com/hibernate/hibernate-orm/commit/f23b550b623a484dd56ccd1028fdaee34c64f907.